ledger/api-server-damlonx/reference/src/main/scala/com/daml/ledger/api/server/damlonx/reference/ReferenceServer.scala

```diff
@@ -21,6 +21,29 @@ import scala.util.Try
 object ReferenceServer extends App {
   val logger = LoggerFactory.getLogger(this.getClass)
 
+  final case class Config(
+      port: Int,
+      portFile: Option[File],
+      archiveFiles: List[File],
+  )
+
+  val argParser = new scopt.OptionParser[Config]("reference-server") {
+    head("DamlOnX Reference Server")
+    opt[Int]("port")
+      .optional()
+      .action((p, c) => c.copy(port = p))
+      .text("Server port. If not set, a random port is allocated.")
+    opt[File]("port-file")
+      .optional()
+      .action((f, c) => c.copy(portFile = Some(f)))
+      .text("File to write the allocated port number to.")
+    arg[File]("<archive>...")
+      .unbounded()
+      .action((f, c) => c.copy(archiveFiles = f :: c.archiveFiles))
+      .text("DAR files to load")
```

Comment:
`DAR files to load. Scenarios are ignored. The servers starts with an empty ledger by default.`